SC_redesign/Crop_and_Soil/manager/field_manager.py

```diff
@@ -0,0 +1,107 @@
+from SC_redesign.Crop_and_Soil.field.field import Field
+from RUFAS.classes import Time, Weather, is_leap_year
+from SC_redesign.Crop_and_Soil.manager.current_weather import CurrentWeather
+from SC_redesign.Crop_and_Soil.manager.output_gatherer import OutputGatherer
+from typing import List, Dict, Optional
+
+"""
+This module is responsible for initializing the `Field` instances that will be simulated and providing an interface to
+the `SimulationEngine` for executing daily and annual routines in the field module.
+"""
+
+
+class FieldManager:
+    def __init__(self, _fields_config: Optional[List[Dict[str, str]]] = None):
+        self.fields: List[Field] = []
+        self.om = OutputGatherer(fields=self.fields)
```

Comment:
```suggestion
        self.fields: List[Field] = []
        self.output_gatherer = OutputGatherer(fields=self.fields)
```
These attributes should have docstrings below them describing what they are.
Also `om` would not be a good name here because that is the abbreviation for the `OutputManager`, so using it here for the `OutputGatherer` would lead to a lot of confusion for someone looking at this code who hadn't already spent a bunch of time working with it.